solr/solrj/src/resources/DeprecatedSystemPropertyMappings.properties

```diff
@@ -55,3 +57,8 @@ solr.ui.enabled=!solr.admin.ui.disabled
 solr.ui.experimental.enabled=!solr.admin.ui.experimental.disabled
 
 solr.zookeeper.chroot.create=create.zk.chroot
+solr.zookeeper.server.confdir=zk.server.conf.dir
+solr.zookeeper.server.datadir=zk.server.data.dir
+solr.zookeeper.server.enabled=zk.run
+solr.zookeeper.server.embedded.host=solr.zk.embedded.host
```

Comment:
ah; only this one did you put "embedded".  again; don't need both "server" and "embedded"; pick one consistently for the set of running and embedded ZK server.

Reply:
I guess i can see an argument for ".server" being for zookeeper server related topics, and then ".embedded" for settings specfic to an embedded ZK...

Reply:
Never mind, I just checked and the mapping file has the correct names.